internal/controller/components/llamastackoperator/monitoring/llamastackoperator-prometheusrules.tmpl.yaml

```
@@ -0,0 +1,98 @@
+              sum(probe_success:burnrate2h{instance=~"llama-stack-k8s-operator-controller-manager"}) by (instance) > (3.00 * (1-0.98000))
+              and
+              sum(probe_success:burnrate1d{instance=~"llama-stack-k8s-operator-controller-manager"}) by (instance) > (3.00 * (1-0.98000))
+            for: 1h
+            labels:
+              severity: warning
+              instance: llama-stack-k8s-operator-controller-manager
+      
+      # RecordingRules for Llama Stack 
+      - name: SLOs - Llama Stack K8s Operator
+        rules:
+        - expr: |
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 10 but found 8

(indentation)